views/js/review/plugins/content/item-answer/item-answer.js

```diff
@@ -110,7 +110,7 @@ define([
     const skippedTab = {
```

Comment:
:warning: I honestly have no idea what is the correct behavior, but this is what looks logical to me. 
We'll need to confirm with QA after the code change is done (I hope at least QA knows how it should be!).

QTI package used by QA: [inf-202_1701860581.zip](https://github.com/oat-sa/extension-lti-test-review/files/13587468/inf-202_1701860581.zip)

### 1. Icon & score-label, both in the panel and in the tab header
  From item properties, which come from testMap in the init action response:  https://lti1p3-platform.playground.kitchen.it.taocloud.org/ltiTestReview/Review/init?serviceCallId=...

_Mapped states:_
- [s-informational] icon: info, score-label: "-"
- [s-pending] icon: clock, score-label: "-"
- [s-correct] icon: green check; score-label: score/maxScore
-  [s-incorrect] icon: red cross; score-label: score/maxScore
-  [s-partial] icon: orange slash; score-label: score/maxScore
-  [s-no-score] icon: empty, score-label: "-"

_Mapping logic:_
```
if (item.informational) {
  [s-informational]
} else if (item.isExternallyScored && item.pendingExternalScore) {
  [s-pending]
} else if (item.maxScore > 0 && item.score > 0 && item.score === item.maxScore) {
  [s-correct]   //PS: it may have item.skipped, but it's still correct, because maybe correct answer is an empty answer (not realistic case though).
} else if (item.maxScore > 0 && item.score === 0 ) {
  [s-incorrect]  //PS: it may have item.skipped, but it's still incorrect, because empty answer is a wrong answer. Was already like this in previous version, and customers didn't request to change that.
} else if (item.maxScore > 0 && item.score > 0 && item.score < item.maxScore) {
  [s-partial]
} else {
  [s-no-score]
  // if item.score=null & item.maxScore=null 
  // if item.score=0 & item.maxScore=0  
}
```

_Difference from previous behavior_:

What is now [s-no-score], before was separated into [s-skipped] and [s-no-score]:
- [s-skipped]
  circle icon  & '0' score-label
  `if ((score=0 & maxScore=0 & skipped=true) || (score=null & maxScore=null & skipped=true))`
- [s-no-score]
  no icon & '-' score-label
  `if ((score=0 & maxScore=0 & skipped=false) || (score=null & maxScore=null & skipped=false))`

Also before for [s-pending], score-label showed score/maxScore. I don't see why it should.

### 2. NO RESPONSE" text in the answer-tab

If `item.skipped === true` 
(So item can have correct icon, because empty response will have maxScore, but still it'll have this text)
And only for "_Your response_" tab, not for "_Correct response_" or _"Informational item_" tabs.

### 3. "Incorrect" filter in the panel
https://github.com/oat-sa/extension-lti-test-review/blob/master/views/js/review/plugins/navigation/review-panel/plugin.js#L51
Should it show [s-incorrect] & [s-partial]?  
Existing code looks closer to [s-incorrect] & [s-partial] & [s-pending] & [s-no-score].

### 4. Which tabs exist
https://github.com/oat-sa/extension-lti-test-review/blob/master/views/js/review/plugins/content/item-answer/item-answer.js#L436
- `if this.getConfig().showCorrect = true`
   * "Your response" -  [s-correct] & [s-incorrect] & [s-partial] & [s-pending] & [s-no-score]
   * "Correct response" - [s-incorrect] & [s-partial] & [s-pending]
      * PS: Are [s-pending] `isExternallyScored`  items even have correct response defined? But whatever, why not show this tab. It was shown before.
   * "Informational item" - [s-informational]
- `if this.getConfig().showCorrect = false`
   * "Your response" -  [s-correct] & [s-incorrect] & [s-partial] & [s-pending] & [s-no-score]
   * "Informational item" -  [s-informational]